src/s6a/handlers/ula_handler.c

```diff
@@ -164,6 +164,12 @@ parse_ula_subscription_data(struct avp *avp_ptr, struct ula_Q_msg *ula)
 						// will finish this part in the following patch
 						// service_slection code is 493
 						// if(g_fd_dict_data.service_slection ==
+						if(apn_cfg_element != NULL)
+						{
+						  	log_msg(LOG_INFO, "Found AVP %d Len = %d \n", apn_cfg_element->avp_code, apn_cfg_element->avp_len);
```

Comment:
if not found avp, should not proceed to access below.

Reply:
Opps..bad check for adding log. In fact that log itself is not required. I added those logs when program was crashing and i had no clue. Since we are going through diameter message, as soon as AVP is traversed we will not even come within this scope.